src/ethereum-providers/index.js

```diff
@@ -29,6 +30,15 @@ const PROVIDERS = new Map(
         'your Ethereum wallet': 'Metamask',
       },
     },
+    {
+      id: 'walletconnect',
+      name: 'WalletConnect',
+      type: 'Mobile',
```

Comment:
Why is it `Mobile`, not `Any`?

Reply:
isn't it supposed to be a mobile wallet?